README.md

```diff
@@ -10,25 +10,27 @@ It uses [Dagster](https://dagster.io) to retrieve and transform several datasour
 
 ### Prerequisites
 
-If you intend to run this project locally via `dagster dev`, you need to have python 3.10 or 3.11 installed. Python 3.12 is not yet supported by dagster. 
+If you intend to run this project locally via `dagster dev`, you need to have python 3.10–3.12 for everything except
```

Comment:
Had to update to Python 3.12 because of relative symlinks: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to (fuuuun!)

Reply:
The switch to Python should be clearly noted in
- the commit doing the switch (which ideally should be isolated from other changes IMO),
- `CHANGELOG.md`,
- the follow-up `ipl-orchestration` PR,
- the `ipl-orchestration` changelog.

Reply:
Done